tests/src/Unit/SocialPostUserTest.php

```diff
@@ -0,0 +1,457 @@
+<?php
+
+namespace Drupal\Tests\social_post\Unit;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\social_post\Entity\SocialPost;
+use Drupal\social_post\SocialPostDataHandler;
+use Drupal\social_post\User\UserManager;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Tests social_post User.
+ *
+ * @group social_post
+ */
+class SocialPostUserTest extends UnitTestCase {
+
+  /**
+   * The tested Social Post UserManager.
+   *
+   * @var \Drupal\social_post\User\UserManager
```

Comment:
Make it `@var \Drupal\social_post\User\UserManager|\PHPUnit_Framework_MockObject_MockObject`, so editors can pick up the `expects` method.

Reply:
This comment was not addressed